actions/transactions/routes.go

```diff
@@ -12,6 +12,29 @@ type Action struct {
 	actions.Action
 }
 
+// NewTransactionsHandler creates the specific package routes in restfull style
+func NewTransactionsHandler(appConfig *config.AppConfig, services *config.AppServices) *routes.Handler {
+	action := &Action{actions.Action{AppConfig: appConfig, Services: services}}
+
+	handler := routes.Handler{
+		BasicEndpointsFunc: routes.BasicEndpointsFunc(func(router *gin.RouterGroup) {
+			basicTransactionGroup := router.Group("/transactions")
+			basicTransactionGroup.GET(":id", action.getByID)
+			basicTransactionGroup.PATCH(":id", action.updateTransaction)
+			basicTransactionGroup.GET("", action.transactions)
+		}),
+		APIEndpoints: routes.APIEndpointsFunc(func(router *gin.RouterGroup) {
+			apiTransactionGroup := router.Group("/transactions")
+			apiTransactionGroup.POST("/drafts", action.newTransactionDraft)
+			apiTransactionGroup.POST("", action.recordTransaction)
+		}),
+		CallbackEndpoints: routes.CallbackEndpointsFunc(func(router *gin.RouterGroup) {
+			router.POST(config.BroadcastCallbackRoute, action.broadcastCallback)
+		}),
+	}
+	return &handler
+}
+
 // NewHandler creates the specific package routes
 func NewHandler(appConfig *config.AppConfig, services *config.AppServices) (routes.BasicEndpointsFunc, routes.OldAPIEndpointsFunc, routes.CallbackEndpointsFunc) {
```

Comment:
This should be renamed to `OldTransactionsHander`

Reply:
Renamed